packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -35,9 +35,6 @@ import { TooltipMixin } from './vaadin-tooltip-mixin.js';
  * -----------------|----------------------------------------
  * `position`       | Reflects the `position` property value.
  *
- * Note: the `theme` attribute value set on `<vaadin-tooltip>` is
```

Comment:
There are still mentions of this in context menu, date picker and date time picker. Should those be removed as well? I think all of those internal components could now be targeted using the theme attribute on the host element?